libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SalesforceSDKManager.m

```diff
@@ -537,7 +538,8 @@ - (void)dismissSnapshot
 
 - (void)clearClipboard
 {
-    if ([SFManagedPreferences sharedPreferences].clearClipboardOnBackground) {
+    
+    if ([SFManagedPreferences sharedPreferences].clearClipboardOnBackground || [SFUserAccountManager sharedInstance].currentUser.idData.shouldDisableExternalPaste) {
```

Comment:
You can directly read the custom attribute here like [here](https://github.com/forcedotcom/SalesforceMobileSDK-iOS/pull/1775/files#diff-5f1af794b53ee821f0831bfae7b4fcb2R120).

It would look like:
```
NSDictionary *customAttributes = [SFUserAccountManager sharedInstance].currentUser.idData.customAttributes;
if (customAttributes) {
    NSString *clearClipboard = customAttributes[kSFDisableExternalPaste];
...
```